conf/untitled.sc

```diff
@@ -0,0 +1,4 @@
+val enrolmentMatcherString = "HMRC-IOSS-ORG~IOSSNumber~"
+val enrolment = "HMRC-IOSS-ORG~IOSSNumber~IM9001234567"
+
+enrolment.substring(enrolmentMatcherString.length)
```

Comment:
Do we need this file?

Reply:
nope, was testing that this worked in a worksheet :joy: